nxcomp/Loop.cpp

```diff
@@ -16685,10 +15495,8 @@ static void handleSetListenersInLoop(fd_set &readSet, int &setFDs)
       setFDs = slaveFD + 1;
     }
 
-    #ifdef DEBUG
-    *logofs << "Loop: Selected listener slaveFD " << slaveFD
+    nxdbg << "Loop: Selected listener slaveFD " << slaveFD
             << " with setFDs " << setFDs << ".\n"
-            << logofs_flush;
-    #endif
+            << std::flush;
   }
 }
```

Comment:
Are you sure you want to have the logging changes in this file part of this PR? Wouldn't it make more sense to leave this PR purely for the new implementation and split logging code changes, also to other files, off as a separate PR?